downloads/platform.md

```diff
@@ -25,7 +25,25 @@ Uninstall Julia by deleting Julia.app and the packages directory in ~/.julia. Mu
 
 ## Linux
 
-It is strongly recommended that the official generic linux binaries from the downloads page be used to install Julia. The following distribution-specific packages are community contributed. They may not use the right versions of Julia dependencies or include important patches that the official binaries ship with. In general, bug reports will only be accepted if they are reproducible on the generic linux binaries on the downloads page.
+It is strongly recommended that the official generic linux binaries from the downloads page be used to install Julia. 
+
+### Generic Linux Binaries
+
+The generic Linux binaries do not require any special installation steps, but you will need to ensure that your system can find the `julia` executable. First, extract the `.tar.gz` file downloaded from the [downloads page](index.html) to a folder on your computer. To run Julia, you can do any of the following:
+
+* Create a symbolic link to `julia` inside a folder which is on your system `PATH`
+* Add Julia's `bin` folder to your system `PATH` environment variable
+* Invoke the `julia` executable by using it's full path, as in `<where you extracted Julia>/bin/julia`
+
+For example, to create a symbolic link to `julia` inside the `/usr/local/bin` folder, you can do the following:
+
+	sudo ln -s <where you extracted the julia archive>/bin/julia /usr/local/bin/julia
```

Comment:
Hm nevermind, it's visible when i click through to looking at the raw source, but github isn't showing it in the source diff. Maybe 4 spaces instead of a tab would show up properly (tabs are generally annoying and shouldn't be used anyway)